bundles/org.openhab.binding.hapero/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,546 @@
+	</channel-type>
+	<channel-type id="furnaceAirDrive" advanced="true">
+		<item-type>Number</item-type>
+		<label>Blower Motor Drive</label>
+		<description>Drive Value of the blower motor</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="furnaceAirO2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Remaining O2</label>
+		<description>Remaining O2 in combustion</description>
+		<state min="0" max="100" step="0.01" pattern="%.2f %%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="storageTemperatureTop">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Top</label>
+		<description>Temperature on top of the storage tank</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state min="0" max="800" step="0.1" pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="storageTemperatureBottom">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Bottom</label>
+		<description>Temperature on the bottom of the storage tank</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state min="0" max="800" step="0.1" pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="temperatureOnSetpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Request Start Temperature</label>
+		<description>Charging of the tank is requested at/below this temperature</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state min="0" max="80" step="0.1" pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="temperatureOffSetpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Request End Temperature</label>
+		<description>Target temperature for charging of the tank</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state min="0" max="80" step="0.1" pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="pumpStatus" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Pump Status</label>
+		<description>Operation status of the pump</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="valveStatus" advanced="true">
+		<item-type>Contact</item-type>
+		<label>Switch Valve Status</label>
+		<description>Current status of the switch valve</description>
+		<category>Contact</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="chargingStatus" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Charging Request</label>
+		<description>Charging of the tank is requested</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Temperature</tag>
+		</tags>
```

Comment:
If it cannot be marked as either measurement or setpoint, I think the tag should be removed.

Reply:
Will tag it as setpoint.